worker/include/RTC/SenderBandwidthEstimator.hpp

```diff
@@ -89,7 +89,7 @@ namespace RTC
 	public:
 		void TransportConnected();
 		void TransportDisconnected();
-		void RtpPacketSent(SentInfo& sentInfo);
+		void RtpPacketSent(const SentInfo& sentInfo);
```

Comment:
Wow unrelated changes hehe

Reply:
Nop :-)

<img width="1430" alt="Screenshot 2023-12-20 at 15 30 07" src="https://github.com/versatica/mediasoup/assets/732340/3a79eeed-2b9d-447a-a290-b4993b338383">

Reply:
This is was a needed change as we are not creating the `onSendCallback()` out of references.